src/lib/ConfigManager.js

```diff
@@ -1,4 +1,5 @@
 "use strict";
```

Comment:
.js files in src should not be in git (only .ts files)
Could we add them to `.gitignore` and remove them?